charter/index.html

```diff
@@ -381,13 +387,11 @@ <h2>Success Criteria</h2>
 
     <!-- Testing and interop -->
 	  <p>In order to advance to
-		  <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each normative specification is expected to have
-		  <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent interoperable
+		  <a href="https://www.w3.org/policies/process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each normative specification is expected to have
+		  <a href="https://www.w3.org/policies/process/#implementation-experience">at least two independent interoperable
 			  implementations</a> of every feature defined in the specification, where
 interoperability can be verified by passing open test suites, and two or
-more implementations interoperating with each other. In order to advance to
-Proposed Recommendation, each normative specification must have an open
-test suite of every feature defined in the specification.</p>
+more implementations interoperating with each other.</p>
```

Comment:
```suggestion
more independent implementations interoperating with each other.</p>
```

Reply:
Considering the context, I like this subtle clarification.